pages/api/v1/acmurl/generate.ts

```diff
@@ -1,11 +1,52 @@
-import type { Data } from "@/util/types";
-import type { NextApiRequest, NextApiResponse } from "next";
+import type {Data} from "@/util/types";
+import type {NextApiRequest, NextApiResponse} from "next";
+import NextCors from "nextjs-cors";
+import {addACMURL, handleExistingACMURL} from "@/util/acmurl";
+import {verifyAuth} from "@/util";
```

Comment:
nit: I think we have eslint on this repo but not prettier, but for consistency I'd prefer this to match the other imports, so just a space between brackets

```ts
import type { Data } from "@/util/types";
```